addons/l10n_fr_fec/security/ir.model.access.csv

```diff
@@ -1,2 +1,2 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_account_fr_fec_create,account.fr.fec.create,model_account_fr_fec,account.group_account_user,1,0,1,0
```

Comment:
@rgo-odoo Can we remove the space access here.  It is weird to use spaces in those names.

Reply:
@jco-odoo done